examples/ecommerce/src/App.vue

```diff
@@ -461,6 +475,17 @@ export default {
         value.max !== null ? value.max : range.max,
       ];
     },
+    openFilters() {
+      document.body.classList.add('filtering');
+      window.scrollTo(0, 0);
+      window.addEventListener('keyup', this.onKeyUp);
+    },
+    closeFilters() {
+      document.body.classList.remove('filtering');
+      const resultsContainer = document.querySelector('.container-results');
```

Comment:
This doesn't need to be computed every time, right? It could be stored outside of the function.

Reply:
4f042f9